data/protein_quantification.csv

```diff
@@ -0,0 +1,27 @@
+,u87pi norm,h5m norm,plate
```

Comment:
Yeah I'm in favor of adding the `sample_id` column as long as the manual edit is tracked by the git history (which it will be).